### ISSUE TITLE
Exception when raising volume above 100% with Mopidy

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/alfred-spotify-mini-player#199]
Getting:

```
stdClass Object
(
    [jsonrpc] => 2.0
    [id] => 1
    [error] => stdClass Object
        (
            [message] => Application error
            [code] => 0
            [data] => stdClass Object
                (
                    [message] => Expected an integer, not u'100'
                    [traceback] => Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/mopidy/internal/jsonrpc.py", line 133, in _handle_single_request
    result = self._unwrap_result(result)
  File "/usr/local/lib/python2.7/site-packages/mopidy/internal/jsonrpc.py", line 222, in _unwrap_result
    result = result.get()
  File "/usr/local/lib/python2.7/site-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/local/lib/python2.7/site-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/local/lib/python2.7/site-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/local/lib/python2.7/site-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/local/lib/python2.7/site-packages/mopidy/core/mixer.py", line 58, in set_volume
    validation.check_integer(volume, min=0, max=100)
  File "/usr/local/lib/python2.7/site-packages/mopidy/internal/validation.py", line 60, in check_integer
    raise exceptions.ValidationError('Expected an integer, not %r' % arg)
ValidationError: Expected an integer, not u'100'

                    [type] => ValidationError
                )

        )

)
2017-08-08 12:47:34|ERROR: invokeMopidyMethod() method: core.mixer.set_volume params: {"volume":"100"} exception:1
```

**Comment by vdesabou** [comment]
Fix available in beta for 7.3 https://github.com/vdesabou/alfred-spotify-mini-player/releases/tag/v7.3-beta

**Comment by vdesabou** [comment]
Version 7.3 is now available http://alfred-spotify-mini-player.com/blog/Version-7.3/
